tutorenrollmentreports/plugin.py

```diff
@@ -12,9 +12,7 @@
 
 config = {
     "add": {
-        "MAIL_TO": [
-            "enrollmentreports@{{ SMTP_HOST }}",
-        ],
+        "MAIL_TO": "[enrollmentreports@{{ SMTP_HOST }},]",
```

Comment:
I understand where this is coming from, but I think this might be the wrong way to fix this. I think you can instead leave this as-is, and instead modify `templates/enrollmentreports/build/enrollmentreports/inventory/config.yml` to include

```yaml
enrollment_report_mail_to: {{ ENROLLMENTREPORTS_MAIL_TO | to_yaml }}
```

But maybe you've tried that already?

Reply:
I tried it now, it dosen't work
`jinja2.exceptions.TemplateAssertionError: No filter named 'to_yaml'.
`

Reply:
Ah, right. That's an Ansible filter, not a built-in Jinja2 filter. Sorry about that. But it turns out that this is made moot by https://github.com/NamrataSitlani/tutor-contrib-enrollmentreports/pull/2 :)

Reply:
Ack.